starter-code/index.html

```diff
@@ -1,31 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
   <title>Instagram</title>
 </head>
 <body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+  <div class = "main">
+  <section class = "background">
+    <div class ="phones">
+      <img src="images/phones.png" alt="phones">
+    </div>
+    <div class="brand">
+      <img src="images/brand.png" alt="brand">
+    </div>
+    <div class = "home">
+    <div class="home-img">
+      <img src="images/home.png" alt="home icon">
+    </div>
+    <div class="log-in">
```

Comment:
No vas nada mal en este apartado, pero recuerda la teoria.
Cuando pones un parrafo, este elemento es de bloque y busca ocupar todo el ancho y salta de linea a los elementos, entonces eso hace que se recorra tu texto de login. lo solucionas de la siguiente manera:

```suggestion
    <div class="log-in">
      <img src="images/home.png" alt="home icon">
      Log in
    </div>
```